platform/linux-generic/odp_timer.c

```diff
@@ -259,6 +259,7 @@ static odp_timer_pool_t timer_pool_new(const char *name,
 
 	if (timer_global.num_timer_pools >= MAX_TIMER_POOLS) {
 		odp_ticketlock_unlock(&timer_global.lock);
+		ODP_ERR("timer pool is full\n");
```

Comment:
This is not an error as such, so I'd use ODP_DBG instead of ODP_ERR. Also, the issue here is not a timer pool being full, but running out of timer pools. A message like `"No more free timer pools\n"` would be more appropriate.

Following the ODP project coding style the commit subject line should be something like:
`linux-gen: timer: add debug print when running out of timer pools`